README.md

```diff
@@ -11,7 +11,11 @@
 [Tips & Tricks](doc/source/notebooks/tips_and_tricks.ipynb)
 
 GPflow is a package for building Gaussian process models in python, using [TensorFlow](http://www.tensorflow.org). It was originally created and is now managed by [James Hensman](http://jameshensman.github.io/) and [Alexander G. de G. Matthews](http://mlg.eng.cam.ac.uk/?portfolio=alex-matthews).
-The full list of [contributors](http://github.com/GPflow/GPflow/graphs/contributors) (in alphabetical order) is Alexander G. de G. Matthews, Alexis Boukouvalas, [Artem Artemev](http://github.com/awav/), Daniel Marthaler, David J. Harris, Hugh Salimbeni, Ivo Couckuyt, James Hensman, Keisuke Fujii, Mark van der Wilk, Mikhail Beck, Pablo Leon-Villagra, Rasmus Bonnevie, ST John, Tom Nickson, Valentine Svensson, Vincent Dutordoir, Zoubin Ghahramani. GPflow is an open source project so if you feel you have some relevant skills and are interested in contributing then please do contact us.
+The full list of [contributors](http://github.com/GPflow/GPflow/graphs/contributors) (in alphabetical order) is
+ Alexander G. de G. Matthews, Alexis Boukouvalas, [Artem Artemev](http://github.com/awav/), Daniel Marthaler, David J
```

Comment:
While I think it's good to keep a list of contributors up to date, I think you would definitely be doing everyone a service if you added everyone.

`git shortlog -s -n` Will give you a list!

Reply:
alternatively `git shortlog -s -n gpflow` will give you contributions to the code in the gpfow directory, though I'm not sure this is necessarily a better list.

Reply:
Sure, I see your point. My understanding was each contributor was responsible for adding its name on the list through a "symbolic" PR, once completed a significant contribution to `develop` over an extended period of time. Personally, I don't think I'm in a position to add any other contributor to the list.

Reply:
alright, makes sense